docs/marketplace/dev/glue-api-guides/202200.0/shopping-lists/managing-shopping-list-items.md

```diff
@@ -0,0 +1,559 @@
+            "sku": "110_19682159"
+        },
+        "links": {
+            "self": "https://glue.mysprykershop.com/shopping-lists/c0bc6296-8a0c-50d9-b25e-5bface7671ce/shopping-list-items/29f1d940-00b6-5492-abf3-d2b5ff15f0b2"
+        }
+    }
+}
+```
+</details> 
+
+<details>
+<summary markdown='span'>Response sample: change the quantity of product offers in the shopping list</summary>
+
+```json
+{
+    "data": {
+        "type": "shopping-list-items",
+        "id": "946451d1-3c40-559e-95c7-ebda2d12bebf",
+        "attributes": {
+            "productOfferReference": "offer3",
+            "merchantReference": "MER000001",
+            "quantity": 10,
+            "sku": "091_25873091"
+        },
+        "links": {
+            "self": "https://glue.mysprykershop.com/shopping-lists/c0bc6296-8a0c-50d9-b25e-5bface7671ce/shopping-list-items/946451d1-3c40-559e-95c7-ebda2d12bebf"
+        }
+    }
+}
+```
+</details> 
+
+For response attributes, see [Add items to a shopping list](#shopping-list-items-response-attributes).
+For the attributes of included resources, see [Retrieve a concrete product](/docs/scos/dev/glue-api-guides/{{page.version}}/managing-products/concrete-products/retrieving-concrete-products.html#concrete-products-response-attributes).
+
+## Remove an item from a shopping list
+
+To remove an item from a shopping list, send the request:
+
+***
+`DELETE` **/shopping-lists/*{% raw %}{{{% endraw %}shopping_list_id{% raw %}}}{% endraw %}*/shopping-list-items/*{% raw %}{{{% endraw %}shopping_list_item_id{% raw %}}}{% endraw %}***
+***
+
+| PATH PARAMETER | DESCRIPTION |
+| --- | --- |
+| ***{% raw %}{{{% endraw %}shopping_list_id{% raw %}}}{% endraw %}*** | Unique identifier of a shopping list to delete an item from. |
+| ***{% raw %}{{{% endraw %}shopping_list_item_id{% raw %}}}{% endraw %}*** | Unique identifier of a shopping list item to remove. To get it, [Retrieve shopping lists](/docs/scos/dev/glue-api-guides/{{page.version}}/managing-shopping-lists/managing-shopping-lists.html#retrieve-shopping-lists), or [Retrieve a shopping list](/docs/scos/dev/glue-api-guides/{{page.version}}/managing-shopping-lists/managing-shopping-lists.html#retrieve-a-shopping-list) with the `shopping-list-items` included. |
+
+### Request
+
+| HEADER KEY | TYPE | REQUIRED | DESCRIPTION |
+| --- | --- | --- | --- |
+| Authorization | string | ✓ | String containing digits, letters, and symbols that authorize the company user. [Authenticate as a company user](/docs/scos/dev/glue-api-guides/{{page.version}}/managing-b2b-account/authenticating-as-a-company-user.html#authenticate-as-a-company-user) to get the value.  |
+
+Request sample:
+
+`DELETE http://glue.mysprykershop.com/shopping-lists/ecdb5c3b-8bba-5a97-8e7b-c0a5a8f8a74a/shopping-list-items/00fed212-3dc9-569f-885f-3ddca41dea08` 
+
+### Response
+
+If the item is removed successfully, the endpoint returns the `204 No Content` status code.
+
+## Possible errors
+
+| CODE | REASON |
```

Comment:
Some of the error codes are missing https://spryker.atlassian.net/wiki/spaces/CORE/pages/226427002/Error+codes#Errorcodes-ShoppingLists